### DISCUSSION TITLE
Norwegian translation

### DISCUSSION THREAD

**Discussion by ringe** [alfio-event/alf.io#1297] [translations]
I'd like to translate Alf.io to Norwegian bokmål nb-no

**Comment by cbellone** [comment]
I have approved your request on POEditor. Thank you for contributing! :+1: 
Please write here as soon as you think the translation is ready to be integrated. Thank you!

**Comment by ringe** [comment]
I only have access to Danish da - would you add Norwegian bokmål nb-no please?

**Reply by cbellone** [reply]
Hi, I have created the entry for nb, but for some reasons I am not able to add you directly.
Can you please request access to the language?

Thanks

**Reply by cbellone** [reply]
Strike that. POEditor support solved the issue :)
You should now have access to the Norwegian Bokmål translation

**Comment by ringe** [comment]
Thanks, I'm in.
